### PR TITLE
Add Travis CI

### PR DESCRIPTION
**Pull request by GBA-17**
Create Travis file and a shell script to run tests.

Currently we test the following:

- Build from root directory
- Create initramfs
- Unit tests for pkg/bootiso

Additional tests can be added by editing `travis.sh`. Note that if this PR is accepted, we still need to enable Travis from their website (otherwise the files are in the repo but never run).

### CODE REVIEW

File: travis.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```
Comment (rminnich) [inline travis.sh:2]: if you put a
set -e
here, it will force an exit on any errors, if that helps. 

Nice to have this back.

Comment (GBA-17) [reply travis.sh:3]: Good idea, I just added `set -e`

**Comment by GBA-17** [comment]
I think I made a mistake when I tried squashing my commits. Somehow some commits from the master branch got added in, so now the PR includes some of the changes that fixed dependency issues.

If you look at the [commit history](https://github.com/u-root/webboot/pull/116/commits) for the PR, the only work that should really be here starts at `Create .travis.yml file`

**Comment by GBA-17** [comment]
I did a hard reset of the branch and manually copied over the files, so the PR should be more accurate now.

**Pull request merged**